clang/lib/Headers/amxcomplexintrin.h

```diff
@@ -62,6 +62,8 @@
 ///    The 2nd source tile. Max size is 1024 Bytes.
 #define _tile_cmmimfp16ps(dst, a, b) __builtin_ia32_tcmmimfp16ps(dst, a, b)
 
+;
```

Comment:
This has been fixed in the latest commit, where the internal function is documented with a Doxygen doc comment so that the `-Wdocumentation` warnings are not triggered.